corehq/apps/userreports/tasks.py

```diff
@@ -15,28 +16,39 @@ def rebuild_indicators(indicator_config_id):
     is_static = indicator_config_id.startswith(StaticDataSourceConfiguration._datasource_id_prefix)
     if is_static:
         config = StaticDataSourceConfiguration.by_id(indicator_config_id)
+        rev = 'static'
     else:
         config = DataSourceConfiguration.get(indicator_config_id)
+        rev = config._rev
         # Save the start time now in case anything goes wrong. This way we'll be
         # able to see if the rebuild started a long time ago without finishing.
         config.meta.build.initiated = datetime.datetime.utcnow()
         config.save()
 
     adapter = IndicatorSqlAdapter(config)
-    adapter.rebuild_table()
 
     couchdb = _get_db(config.referenced_doc_type)
-    relevant_ids = get_doc_ids_in_domain_by_type(config.domain, config.referenced_doc_type,
-                               database=couchdb)
+    client = get_redis_client().client.get_client()
```

Comment:
are the needed methods not exposed on the regular RedisCache object? I've seen weird behavior in the past where interacting with the regular RedisCache object and interacting with the raw client produces different results, so would be good to avoid using the raw client if possible.